execution/executor-benchmark/src/lib.rs

```diff
@@ -98,10 +101,48 @@ impl TransactionGenerator {
     }
 
     fn run(&mut self, init_account_balance: u64, block_size: usize, num_transfer_blocks: usize) {
+        self.gen_account_creations(block_size);
         self.gen_mint_transactions(init_account_balance, block_size);
         self.gen_transfer_transactions(block_size, num_transfer_blocks);
     }
 
+    fn gen_account_creations(&self, block_size: usize) {
+        let assoc_account = association_address();
+
+        for (i, block) in self.accounts.chunks(block_size).enumerate() {
+            let mut transactions = Vec::with_capacity(block_size);
+            for (j, account) in block.iter().enumerate() {
+                let txn = create_transaction(
+                    assoc_account,
+                    (i * block_size + j) as u64,
+                    &self.genesis_key,
+                    self.genesis_key.public_key(),
+                    encode_create_parent_vasp_account(
+                        lbr_type_tag(),
+                        account.address,
+                        account.auth_key_prefix(),
+                        b"testnet".to_vec(),
+                        b"https://libra.org".to_vec(),
+                        // A bogus (but valid ed25519) compliance public key
+                        vec![
+                            0xb7, 0xa3, 0xc1, 0x2d, 0xc0, 0xc8, 0xc7, 0x48, 0xab, 0x07, 0x52, 0x5b,
```

Comment:
How are these bytes generated? Maybe include some comments for regenerating such bytes?

Reply:
These are just random valid bytes that I found on Google. It doesn't need to be anything special, just a valid ed25519 public key is all.